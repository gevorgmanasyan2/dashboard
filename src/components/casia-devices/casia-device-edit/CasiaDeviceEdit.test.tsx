/* eslint-disable no-undef */
import React from 'react'
import { shallow } from 'enzyme'
import CasiaDeviceEdit from './CasiaDeviceEdit'

it('should render CasiaDeviceEdit component', () => {
  expect(shallow(<CasiaDeviceEdit />)).toMatchSnapshot()
})
