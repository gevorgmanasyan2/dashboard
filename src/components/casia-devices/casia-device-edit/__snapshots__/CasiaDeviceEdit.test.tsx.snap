// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CasiaDeviceEdit component 1`] = `
<article
  className="casia__device__edit"
>
  <section
    className="casia__device__edit__list"
  >
    <div
      className="casia__device__edit__list__items"
    >
      <h3>
        Casia Devices
      </h3>
      <ul>
        <li>
          Casia Long Range, A3, Under maintenance
        </li>
        <li>
          Casia Long Range, A9, Active
        </li>
        <li>
          Casia 360, A3, Active
        </li>
      </ul>
    </div>
    <div
      className="casia__device__edit__list__form"
    >
      <div
        className="casia__device__edit__list__form--title"
      >
        <h3>
          Edit Casia Device
        </h3>
        <p>
          Serial #: H234 12348970123
        </p>
      </div>
      <ShareableInput
        label=""
        placeholder="Name"
      />
      <ShareableTextarea
        label="Description"
      />
      <ShareableInput
        label="Registration #"
      />
      <ShareableSelect
        label="Installed on UAS"
      />
      <ShareableSelect
        label="Status"
      />
    </div>
  </section>
  <section
    className="casia__device__edit__buttons"
  >
    <Link
      to="/casiadevice-form"
    >
      <ActionButtons
        buttonText="Back"
        className="config__container__buttons--cancel"
      />
    </Link>
    <ActionButtons
      buttonText="Save"
      className="config__container__buttons--next"
    />
  </section>
</article>
`;
